src/extension.ts

```diff
@@ -26,11 +32,18 @@ export function activate(context: ExtensionContext) {
       // Register the server for java
       documentSelector: ['xml'],
       revealOutputChannelOn: RevealOutputChannelOn.Never,
-      initializationOptions: {"lsp4xml.logPath": logfile}
+      initializationOptions: {"lsp4xml.logPath": logfile},
```

Comment:
It might not be required now, but eventually you might want to send the settings as part of initializationOptions, if you need specific settings *before* handling the didChangeConfiguration request which is sent after the server has initialized.
Just keep that in mind for now.